### ISSUE TITLE
[Rule] testNoEquivalentSuperclassMethods should explicitely tell what methods are problematic

### ISSUE THREAD

**Issue by demarey** [guillep/pharo-core#17334]
We have this test that is bad because:
- if we remove redundant methods (accepted) and introduce new ones, we won’t get warned
- it does not explicitly display a message with the faulty methods

It should be updated by using a list of currently accepted redudant methods and provide a nice message telling the user what methods are problematic

``` smalltalk
testNoEquivalentSuperclassMethods

	| methods |
	"we do not care about methods that are installed from traits"
	methods := SystemNavigation new allMethods reject: [ :method | method isFromTrait ].

	methods := methods select: [ :method |
		           method overriddenMethod
			           ifNotNil: [ :overridenMethod | method equivalentTo: overridenMethod ]
			           ifNil: [ false ] ].

	self assert: methods size <= 326
```

**Comment by MarcusDenker** [comment]
yes, I wanted to actively kill the methods but dropped the ball (I did the same before e.g. for unused variables)

To see how the tests should look like, the unused vars one show how to add a list of known failures (e.g. from test data).

326 is a bit too much for such a whitelist, though.

A lot of the remaining are methods with pragmas that are copied down, these I think an not just be removed.
